src/webview/WebViewScreen.tsx

```diff
@@ -195,21 +204,27 @@ function WebViewScreen({ route, navigation }: Props) {
 
   return (
     <SafeAreaView style={styles.container} edges={['bottom']}>
-      <WebView
-        ref={webViewRef}
-        originWhitelist={['https://*', 'celo://*']}
-        onShouldStartLoadWithRequest={handleLoadRequest}
-        source={{ uri }}
-        startInLoadingState={true}
-        renderLoading={() => <ActivityIndicator style={styles.loading} size="large" />}
-        onNavigationStateChange={(navState) => {
-          setCanGoBack(navState.canGoBack)
-          setCanGoForward(navState.canGoForward)
-          handleSetNavigationTitle(navState.url, navState.title, navState.loading)
-        }}
-        mediaPlaybackRequiresUserAction={mediaPlaybackRequiresUserAction}
-        testID={activeDapp ? `WebViewScreen/${activeDapp.name}` : 'RNWebView'}
-      />
+      <KeyboardAvoidingView
+        style={styles.keyboardView}
+        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
```

Comment:
Is this really needed on iOS? I just tried quickly on iOS and inputs are already avoiding the keyboard.

Reply:
Do you mean the behavior or the whole `KeyboardAvoidingView`?

Reply:
I don't think it's needed on iOS, but I also don't see any detriment in adding it. I tested quickly on iOS by using Withdraw > Spend > Bidali > Amazon Gift Card > Custom Amount.

Reply:
Sounds good, thanks for the details 👍